frontend/src/Series/Details/SeriesDetailsLinks.css

```diff
@@ -1,20 +1,72 @@
 .links {
-  margin: 0;
+  display: flex;
+  flex-wrap: wrap;
+  gap: 1rem;
+  justify-content: flex-start;
+  align-items: flex-start;
+  margin-right: 4px;
+}
+
+.linkGroup {
+  display: flex;
+  flex-direction: column;
+  align-items: left;
+  text-align: left;
 }
 
 .link {
   white-space: nowrap;
+  width: 100%;
 }
 
 .linkLabel {
   composes: label from '~Components/Label.css';
-
   cursor: pointer;
+  width: 100%;
+}
+
+.idLabel {
+  margin-left: 4px;
+}
+
+.idCopyButton {
+  border-radius: 4px !important;
+  padding: 2px 6px !important;
+  /* margin-left: 6px !important; */
+  margin-right: 0 !important;
+  margin-left: auto !important;
 }
```

Comment:
This should be addressed via a different component for the button, instead of important overriding.